spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'Guest Show page' do
+
+  before do 
+    @hotel_1 = Hotel.create!(name: 'Stanly Hotel', location: 'Estes Park')
+    @room_401 = @hotel_1.rooms.create!(rate: 50, suite: 'Room 401')
+    @room_402 = @hotel_1.rooms.create!(rate: 50, suite: 'Room 402')
+    @guest_1 = Guest.create!(name: 'Jeff', nights: 1)
+    GuestRoom.create(guest: @guest_1, room: @room_401)
+  end
+  describe 'As a user' do
+    describe '/guests/:id' do
+      it 'I see the guests name ' do
+        visit "/guests/#{@guest_1.id}"
+
+        expect(page).to have_content("Jeff's page")
+      end 
+
+      it 'And I see a list of all the rooms they have stayed in as well as room fields and hotel' do
+        visit "/guests/#{@guest_1.id}"
+
+        expect(page).to have_content("Rooms stayed in")
+        expect(page).to have_content("Rooms: Room 401")
+        expect(page).to have_content("Rate: 50")
+        expect(page).to have_content("Hotel: Stanly Hotel")
+      end
```

Comment:
The only thing I would add here is some `to_not` assertions as well to make sure the other room doesn't show up. This makes the test a little more robust.

```ruby
expect(page).to_not have_content("Rooms: Room 402")
```

You can also consider adding some `within` blocks as well:

```ruby
within "#rooms" do
  expect(page).to have_content("Rooms stayed in")
  expect(page).to have_content("Rooms: Room 401")
  expect(page).to have_content("Rate: 50")
  expect(page).to have_content("Hotel: Stanly Hotel")
  expect(page).to_not have_content("Rooms: Room 402")
end
```